styles/variables.less

```diff
@@ -7,3 +7,13 @@
 @gh-background-color-red: #dc3545;
 @gh-background-color-purple: #6f42c1;
 @gh-background-color-green: #28a745;
+
+
+// diff colors -----------------
+// Needs to be semi transparent to make it work with selections and different themes
+
+@github-diff-deleted:           fade(hsl(353, 100%, 66%), 15%); // similar to .com's hsl(353, 100%, 97%)
```

Comment:
so cool.  I really want to learn more about css colors.

Reply:
If possible, it's best to only use the [UI](https://github.com/atom/atom/blob/master/static/variables/ui-variables.less) and [syntax](https://github.com/atom/atom/blob/master/static/variables/syntax-variables.less) theme variables. Then it _should_ look ok in all themes.

But sometimes making an exception is fine too. :grimacing: In this case, it felt better to use the .com colors for the diff and not "success" and "error". But it needed to be semi transparent to make the selections visible underneath. Less has a bunch of functions to tweak colors, like [fade()](http://lesscss.org/functions/#color-operations-fade) to make an existing color more transparent.

I also have been using [mix()](http://lesscss.org/functions/#color-operations-mix) sometimes. For example 

```less
.class {
  background-color: mix(@text-color, @base-background-color, 10%);
}
```

mixes `10%` of the text color with the background color -> making the new background color slightly darker with light themes or slightly lighter in dark themes.